harvard-lts/drs-migration-dashboard
# Working prototype.

**This is a working prototype. It uses static data to display two piecharts.**
* * *

* See: https://www.chartjs.org/docs/latest/

# What does this Pull Request do?
This builds a simple Flask application. It uses a replica of the design from the COVIDverify project. It has a single page right now that displays two piecharts.

# How should this be tested?

A description of what steps someone could take to:
* Follow the README instructions to spin up the application
* Go to https://localhost:3001/migrationstatus/piechart/ to see the two pie charts.

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? No 
- integration tests? No

# Interested parties
@dl-maura @awoods @enriquediaz
